### DISCUSSION TITLE
Using the latest Snapshot version of Quarkus via Maven

### DISCUSSION THREAD

**Discussion by mr-mos** [quarkusio/quarkus#36844] [Community]
I generated a new Quarkus application via https://code.quarkus.io.
How do I need to change the` pom.xml `to work with the latest SNAPSHOT version of Quarkus?
Thanks!

**Comment by mschorsch** [comment]
https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#using-snapshots
